qlib/workflow/task/collect.py

```diff
@@ -139,6 +139,7 @@ def __init__(
         rec_filter_func=None,
         artifacts_path={"pred": "pred.pkl"},
         artifacts_key=None,
+        filter_string: str = "",
```

Comment:
Qlib's framework is decoupled with mlflow.
So these filter_string specific things shouldn't appear here.
I think `list_kwargs={}`  and `recs = self.experiment.list_recorders(**self.list_kwargs)` will make it more  loosely-coupled

Reply:
fixed as recommend :)